README.md

```diff
@@ -91,6 +91,18 @@ RULES_SCHEMA 		| Specify the rules schema for your database. | `rules`
 DATA_SCHEMA 		| Specify the data schema for your database. | `data`
 CUSTOMERDATA_SCHEMA | If configured in your database, set the customer data schema for your database. If you do not provide a value, this setting defaults to `dataSchema`. |
 
+### Custom artifactory settings used for downloading JDBC driver
+
+You can specify custom artifactory authentication details if it is secured with Basic or APIKey Authentication which is used for downloading JDBC driver.
```

Comment:
Configure the Basic or APIKey Authentication details for a custom artifactory authentication you use for managing a JDBC driver for your organization.